lib/tasks/test.rake

```diff
@@ -0,0 +1,9 @@
+# lib/tasks/test.rake
+namespace :test do
+  namespace :system do
+    task :with, [:driver] => :environment do |task, args|
```

Comment:
Add a description for the rake task otherwise it won't be listed when doing `rake -T`